django/utils/log.py

```diff
@@ -87,6 +87,7 @@ def __init__(self, include_html=False, email_backend=None):
         super().__init__()
         self.include_html = include_html
         self.email_backend = email_backend
+        self.reporter_klass = get_default_exception_reporter()
```

Comment:
Rather than adding the method call here, add a kwarg, as per the suggestion in Tim's comment. This can then be provided in the `LOGGING` config. 

See `logging.txt` in the `Handlers` section for the examples of `include_html` and `email_backend`. Similar docs are needed for the new parameter.